code/modules/mob/living/silicon/robot/component.dm

```diff
@@ -98,25 +96,21 @@
 	name = "radio"
 	external_type = /obj/item/robot_parts/robot_component/radio
 	energy_consumption = 1
-	max_damage = 40
```

Comment:
Unatomic change, made radio's health drop from 40 to 15

Reply:
intentional